File: LibraryManagementSystem/Backend/LibraryManagementSystem/LibraryManagementSystem.Entities/Models/Library/Book.cs
namespace LibraryManagementSystem.Entities.Models.Library
{
    public class Book
    {
        public int BookId { get; set; }
        public string? ISBN10 { get; set; }
        public string? ISBN13 { get; set; }
        public string? Classification { get; set; }
        public string? Title { get; set; }
        public string? Description { get; set; }
        public short PublicationYear { get; set; }
        public byte[]? Image { get; set; }
        public bool IsActive { get; set; }
        public int PublisherId { get; set; }
        public int CategoryId { get; set; }
        public short NumberOfCopies { get; set; }
        public short BorrowedCopies { get; set; }
        public bool IsAvailable { get; set; }
    }
}

File: LibraryManagementSystem/Frontend/library-management-system-web/src/app/entities/models/library/book.ts
export class Book {
    bookId: number | undefined;
    iSBN10: string | null | undefined;
    iSBN13: string | null | undefined;
    classification: string | null | undefined;
    title: string | null | undefined;
    description: string | null | undefined;
    publicationYear: number | undefined;
    image: string | null | undefined;
    isActive: boolean | undefined;
    publisherId: number | undefined;
    categoryId: number | undefined;
    numberOfCopies: number | undefined;
    borrowedCopies: number | undefined;
    isAvailable: boolean | undefined;
}
